plugin/ochttp/trace.go

```diff
@@ -158,3 +149,72 @@ func responseAttrs(resp *http.Response) []trace.Attribute {
 		trace.Int64Attribute(StatusCodeAttribute, int64(resp.StatusCode)),
 	}
 }
+
+func status(statusCode int) trace.Status {
+	var code int32
+	if code < 200 || code >= 400 {
+		code = codeUnknown
```

Comment:
HTTP 1XX are not errors

Reply:
HTTP 100s are intermediate status codes, and they shouldn't be final.

For example, HTTP 100 behavior as explained at https://tools.ietf.org/html/rfc7231#section-5.1.1:

A server that sends a 100 (Continue) response MUST ultimately send
      a final status code, once the message body is received and
      processed, unless the connection is closed prematurely.